homeassistant/components/camera/mjpeg.py

```diff
@@ -74,6 +74,10 @@ def __init__(self, device_info):
         self._mjpeg_url = device_info[CONF_MJPEG_URL]
         self._still_image_url = device_info.get(CONF_STILL_IMAGE_URL)
 
+        logging.getLogger("urllib3.connectionpool").addFilter(
```

Comment:
Hmm merged it now but now I realize that this is going to add the filter many times, once for each camera…

Reply:
this should be added to setup platform and check that it only adds it once.

Reply:
Should be a pretty easy change. I'll get an updated PR up for it.

Reply:
new PR with changes #17050